### DISCUSSION TITLE
Can the CWatM model simulate river infiltration recharge to groundwater?

### DISCUSSION THREAD

**Discussion by Chengwenju** [iiasa/CWatM#101] [Q&A] [answered]
hello, @mikhailsmilovic 

Can the CWatM model simulate river infiltration recharge to groundwater?

**Comment by dof1985** [comment, marked as answer]
Hello @Chengwenju 

CWatM can simulate river and lake infiltration into groundwater using the Modflow extension.
I hope it helps.

**Reply by Chengwenju** [reply]
thank you so much
